### ISSUE TITLE
get MirrorBot working with startGamebotsDM11AlbatrossServer

### ISSUE THREAD

**Issue by friedmaa** [schrum2/MM-NEAT#141]
Right now I can boot up the server, but when I try to get MirrorBot to run it it, it crashes with the message

```
ComponentCantStartException[UT2004Bot[MirrorBot]: Can't start: BusAwareCountDownLatch: Interrupted because bus was stopped (fatal error, or watched component stopped) while waiting on the latch. (caused by: BusAwareCountDownLatch: Interrupted because bus was stopped (fatal error, or watched component stopped) while waiting on the latch.)]
Aborting...
	at cz.cuni.amis.pogamut.base.agent.impl.AbstractAgent.start(AbstractAgent.java:474)
	at cz.cuni.amis.pogamut.base.agent.utils.runner.impl.AgentRunner.startAgent(AgentRunner.java:515)
	at cz.cuni.amis.pogamut.base.agent.utils.runner.impl.AgentRunner.startAgentWithParamsMain(AgentRunner.java:365)
	at cz.cuni.amis.pogamut.base.agent.utils.runner.impl.AgentRunner.startAgent(AgentRunner.java:189)
	at cz.cuni.amis.pogamut.ut2004.utils.UT2004BotRunner.startAgent(UT2004BotRunner.java:195)
	at fr.enib.mirrorbot4.MirrorBot4.main(MirrorBot4.java:190)
Caused by: BusStoppedInterruptedException[BusAwareCountDownLatch: Interrupted because bus was stopped (fatal error, or watched component stopped) while waiting on the latch.]
	at cz.cuni.amis.pogamut.base.component.bus.event.BusAwareCountDownLatch.checkBusStop(BusAwareCountDownLatch.java:173)
	at cz.cuni.amis.pogamut.base.component.bus.event.BusAwareCountDownLatch.await(BusAwareCountDownLatch.java:167)
	at cz.cuni.amis.pogamut.ut2004.bot.impl.UT2004Bot.startAgent(UT2004Bot.java:227)
	at cz.cuni.amis.pogamut.base.agent.impl.AbstractAgent.start(AbstractAgent.java:460)
	... 5 more

```

**Comment by friedmaa** [comment]
PUT ON HOLD BECAUSE IT'S WEIRD

**Comment by friedmaa** [comment]
It decided to work today
